forest/src/cli/auth_cmd.rs

```diff
@@ -57,7 +57,7 @@ impl AuthCommands {
                     Ok(token) => {
                         let mut addr = Multiaddr::empty();
                         addr.push(Protocol::Ip4(DEFAULT_HOST.parse().unwrap()));
```

Comment:
in general, shouldn't this be overridden as well?

Reply:
It should. Fixed.